packages/generators/utils/packageExists.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * packageExists
+ *
+ * Check whether the given package exist or not
+ */
+
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname( fileURLToPath( import.meta.url ) );
+
+const packagePath = path.join( __dirname, '../../' );
+
+const packageNames = fs.readdirSync( packagePath );
+
+const packageExists = ( packageName ) => packageNames.includes( packageName );
```

Comment:
I wonder if this should be a bit more clever, though. Yes, many of the packages exist in the monorepo, and the package name matches the package dir name, however, that's not always the case. There are a bunch of `@automattic` packages that are outside of the repo and within the `Automattic` GitHub organization. Should we make and maintain a list of them and ensure that if someone wants to create a package, it doesn't exist already in the GitHub org (therefore in npm)?

Reply:
Follow-up for that sounds good 👍 Thanks!